app/scripts/components/common/chip/chip.component.scss

```diff
@@ -0,0 +1,110 @@
+.chip {
+  display: inline-block;
+  padding: 4px 8px;
+  border: 1px solid #E5E5E5;
+  border-radius: 16px;
```

Comment:
let's use rem instead. Also, it will be nice to store these values as a common variables. (for future possible design changes)